src/Maestro/Maestro.ContainerApp/Queues/WorkItems/CheckDailySubscriptionsWorkItem.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Maestro.ContainerApp.Queues.WorkItems;
+
+public class CheckDailySubscriptionsWorkItem : BackgroundWorkItem
```

Comment:
You also need to register the child class in `BackgroundWorkItem` via an attribute